MetaMask/metamask-mobile
# Clicking toAddress to add it to address book

Fixes: #6008 

A reusable component is created to wrap child elements, the child elements when clicked will trigger add-to-address-book modal to open.

QA steps:
Add to address book works well at:
- send page where to address is selected
- confirm send page
- any DAPP transaction
